Update/kage_003.txt

```diff
@@ -427,39 +427,39 @@ void main()
-			NULL, " Besides, it can be difficult to get answers through normal questioning because the people in this city tend to be rather reserved.\"", Line_Normal);
+			NULL, "Trust me, I know the people in this town can be pretty frank about what they think, but if you don't know the right questions to ask they can be difficult enough to make you want to pull your hair out.\"", Line_Normal);
 	ClearMessage();
 
 //r…かつての雛見沢のように排他的な結束がある場所の捜査は、聞き込みなどの情報収集において大きな障害がある。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "…かつての雛見沢のように排他的な結束がある場所の捜査は、聞き込みなどの情報収集において大きな障害がある。",
-			NULL, "...Investigating a place like Hinamizawa with a fiercely independent local identity can also be a major roadblock to information gathering.", Line_Normal);
+			NULL, "Investigating such a tightknit community as Hinamizawa is huge a problem for law enforcement. People are fiercely loyal to eachother above all, which serverely limits the amount of information we're able to gather there.", Line_Normal);
 	ClearMessage();
 
 //r逆に新興の市街によくある、隣人に無関心な雰囲気も警察にとっては厄介な場合が多い。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "逆に新興の市街によくある、隣人に無関心な雰囲気も警察にとっては厄介な場合が多い。",
-			NULL, "However, the atmosphere in developing towns can also be troublesome for the police, as the people feel completely indifferent toward their neighbors.", Line_Normal);
+			NULL, "But on the other hand, the atmosphere in newly founded communities also has its own issues. When people feel completely indifferent toward their neighbors, they might not pay enough attention to even know any meaningful information in the first place.", Line_Normal);
 	ClearMessage();
 
 //r各人から集めた情報が非常に曖昧で、正確性に欠けるものとなる場合が多いからだ。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "各人から集めた情報が非常に曖昧で、正確性に欠けるものとなる場合が多いからだ。",
-			NULL, "The information gained from each person can be very ambiguous, and in some cases even inaccurate.", Line_Normal);
+			NULL, "There's been plenty of times where information gained from each witness can be very ambiguous, and in some cases even inaccurate.", Line_Normal);
 	ClearMessage();
 
 //赤坂rvS08/13/321000317.「…あと、よそ者の刑事が好き勝手に捜査を行えば、そちらの刑事課さんにとっても色々と問題があるかと思います」
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#797d8a>赤坂</color>", NULL, "<color=#797d8a>Akasaka</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 13, "ps3/s08/13/321000317", 256, TRUE);
 	OutputLine(NULL, "「…あと、よそ者の刑事が好き勝手に捜査を行えば、そちらの刑事課さんにとっても色々と問題があるかと思います」",
-			NULL, "\"...And if we perform an arbitrary investigation as outsiders, it could pose problems for the local criminal investigation division as well.\"", Line_Normal);
+			NULL, "\"And if people think that we're just a couple of outside police officers sticking our nose into their business, it could pose problems for the local detectives as well.\"", Line_Normal);
```

Comment:
```suggestion
			NULL, "\"And if people think that we're just a couple of outside police officers sticking our noses into their business, it could pose problems for the local detectives as well.\"", Line_Normal);
```